dune

```diff
@@ -15,7 +15,10 @@
   lev-fiber-csexp
   (library
    (name lev_fiber_csexp)
-   (libraries fiber csexp stdune dyn lev_fiber))
```

Comment:
We don't need to instrument anything under `(subdir vendor`.

Reply:
Yeah this was a rough PR so I could see how interacted in the CI. I will tighten it up later.